### PR TITLE
Replace JUnit 3 imports with JUnit Jupiter

### PR DESCRIPTION
**Pull request by stefanbirkner**
JUnit 3 is not maintained for years while JUnit Jupiter as part of JUnit
5 already exists for for than four years.

### CODE REVIEW

File: docs/mobile-apps/automated-testing/appium.md [hunk 1]
```diff
@@ -359,8 +359,7 @@ Because of the wide variations in recording technique, this beginning section om
 The following example runs a test by asserting that the login response message is equal to an expected success message:
 
 ```java
-import junit.framework.Assert;
-import junit.framework.TestCase;
```
Comment (stefanbirkner) [inline docs/mobile-apps/automated-testing/appium.md:363]: The import `junit.framework.TestCase` was not used in the snippet at all.

**Review by kimsauce — APPROVED** [review]

**Comment by titusfortner** [comment]
We should update it to JUnit 4.
We need to be very careful about promoting JUnit 5 as long as it doesn't work with Selenium 4.

**Comment by sweeneyskirt-sl** [comment]
Looks like these changes were merged in another PR, so I am closing this one.

**Comment by titusfortner** [comment]
Well... JUnit 5 is fixed with Selenium beta 4 and I didn't realize it, so now I can tell people to start using it again. So... Thanks and sorry?

**Pull request closed without merging**